docs/app/app.component.html

```diff
@@ -1,8 +1,22 @@
-<uxd-navigation-bar></uxd-navigation-bar>
+<!-- Documentation header -->
```

Comment:
When I first navigate from the Showcase Overview page to the partition map the footer remains visible:

![image](https://user-images.githubusercontent.com/20795331/72336539-e27dd080-36b8-11ea-9d53-9ab554b29867.png)

Inversely if I am on the partition map and navigate back to the showcase page the header remains visible:

![image](https://user-images.githubusercontent.com/20795331/72336620-06411680-36b9-11ea-8dd6-0ef9626c1592.png)

Reply:
Sorry must have committed the wrong code, had this working a minute ago.